src/tree/AttachedAccountsTreeItem.ts

```diff
@@ -64,20 +64,24 @@ export class AttachedAccountsTreeItem extends AzureParentTreeItem {
     }
 
     public static validateMongoConnectionString(value: string): string | undefined {
+        value = value.trim();
+
         if (value && value.match(/^mongodb(\+srv)?:\/\//)) {
             return undefined;
         }
+
         return MONGO_CONNECTION_EXPECTED;
     }
 
     private static validateDocDBConnectionString(value: string): string | undefined {
+        value = value.trim();
```

Comment:
This repo doesn't have stictNullChecks turned on so you have to be extra careful. I'm pretty sure `value` could be undefined and you should handle that appropriately